Makefile

```diff
@@ -122,12 +124,18 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build --build-arg ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS} -t ${IMG} .
 
 .PHONY: docker-push
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+install-cert: ## Install cert manager for webhooks
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$(CERT_MANAGER_VERSION)/cert-manager.yaml
```

Comment:
Could you not use jetstack and use a more recent version? It should work. I can test it on crc on Monday.

Reply:
I was going to ask you about if you wanted to blindly copy jetstack or use a real one on Monday since I assumed you'd be gone

Reply:
👀